### DISCUSSION TITLE
Add Fairy Ring Text to 'Donovan the Family Handyman' Medium Clue Help

### DISCUSSION THREAD

**Discussion by trelkins3** [runelite/runelite#15358] [Ideas]
The helper text for the easy clue to talk to Louisa at Sinclair Mansion indicates that fairy ring CJR should be used. The helper text for medium clue "Speak to Donovan, the Family Handyman" indicates that Donovan is at Sinclair Mansion, but does not mention fairy ring CJR.

I suggest changing the Donovan helper text to mention fairy ring CJR for player convenience.

**Comment by Nightfirecat** [comment]
I've created #15366 for implementing this. Thanks for the suggestion!
